cpp/arcticdb/storage/storage.hpp

```diff
@@ -73,6 +73,13 @@ class KeyNotFoundException : public ArcticSpecificException<ErrorCode::E_KEY_NOT
             keys_(std::make_shared<Composite<VariantKey>>(std::move(keys))) {
     }
 
+    explicit KeyNotFoundException(const VariantKey& single_key):
+            KeyNotFoundException(Composite<VariantKey>{VariantKey{single_key}}) {}
```

Comment:
For reference this is a [delegating constructor](https://en.cppreference.com/w/cpp/language/constructor#:~:text=member%20initializer.-,Delegating%20constructor,-If%20the%20name).